src/actions/clusterActions.js

```diff
@@ -15,6 +15,9 @@ const clustersApi = new GiantSwarm.ClustersApi();
 function clustersLoadArrayToObject(clusters, provider) {
   return clusters
     .map(cluster => {
+      // Remove cluster's create_date because we are loading it in cluster details call
+      delete cluster.create_date;
```

Comment:
This is because the `create_date` field in the clusters list and cluster details APIs response have different time format, one has seconds rounded and the other has nanosedconds I believe. 

As they are different strings, we update the value each time `clustersList()` and `clusterLoadDetails()` run. Then the selector can't memoize because it is receving different inputs every time this calls are performed, so it recomputes after every single call.

We could format the two strings to be exactly the same value, or modify responses in the API repo to use the same format, but as we don't need this to be present during initialization I just removed the key and let the cluster detail action to populate it in store.